cypress/cypress.config.js

```diff
@@ -19,13 +19,19 @@ module.exports = defineConfig({
         }
       })
     },
-    supportFile: 'cypress/support/e2e.js',
-    specPattern: 'cypress/e2e/*.cy.{js,jsx,ts,tsx}'
+    supportFile: 'support/e2e.js',
+    specPattern: 'e2e/*.cy.{js,jsx,ts,tsx}'
   },
   env: {
     host: process.env.OPENSHIFT_HOST || 'https://console-openshift-console.apps.ocp-edge-cluster-0.qe.lab.redhat.com',
     auth: process.env.OPENSHIFT_AUTH || 'https://oauth-openshift.apps.ocp-edge-cluster-0.qe.lab.redhat.com',
     username: process.env.OPENSHIFT_USERNAME || 'kubeadmin',
-    password: process.env.OPENSHIFT_PASSWORD || 'kubeadmin'
+    password: process.env.OPENSHIFT_PASSWORD || 'kubeadmin',
+    image: process.env.QUAY_IMAGE || 'quay.io/sdelacru/flightctl-centos:v1',
+    os: process.env.PLATFORM || 'Linux',
+    arch: process.env.ARCH || 'x86_64',
+    repository: process.env.REPOSITORY || 'https://github.com/flightctl/flightctl-demos',
+    revision: process.env.REVISION || 'main',
```

Comment:
Same as in the other comment, I suggest to use a specific branch for the e2e tests so that they cannot be broken by mistake.